docs/doc/14-sql-commands/00-ddl/103-aggregating-index/refresh-aggregating-index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: REFRESH AGGREGATING INDEX
+---
+
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.151"/>
+
+import EEFeature from '@site/src/components/EEFeature';
+
+<EEFeature featureName='AGGREGATING INDEX'/>
+
+Refreshes an aggregating index to update its stored results. Databend recommends refreshing an aggregating index before executing a query that relies on it to retrieve the most up-to-date data.
+
+:::note
+When using Databend Cloud, manual execution of this refresh command is unnecessary, as the system automatically handles index updates for you.
+:::
+
+## Syntax
+
+```sql
+REFRESH AGGREGATING INDEX <index_name>
```

Comment:
refresh has an optional `limit`, eg: `refresh aggregating index xxx limit 10` to refresh 10 blocks since the last refresh.
And we highly recommend doing refresh with a limit to reduce memory usage.